src/extension.ts

```diff
@@ -105,8 +106,8 @@ async function getAndSetNewGlobalState(context: vscode.ExtensionContext) {
   let logger: Logger | undefined = undefined;
   let backend: ExtensionBackend | undefined = undefined;
   try {
-    outputChannel = vscode.window.createOutputChannel("dprint");
-    logger = new Logger(outputChannel);
+    outputChannel = DprintOutputChannel.getOutputChannel();
```

Comment:
There's a small possibility here that the outputChannel gets crated as a singleton, but then disposed below and so the next time the extension is activated it would use a singleton that's disposed.